setup.cfg

```diff
@@ -41,6 +42,8 @@ invenio_base.apps =
     invenio_notifications = invenio_notifications:InvenioNotifications
 invenio_base.api_apps =
     invenio_notifications = invenio_notifications:InvenioNotifications
+invenio_base.api_blueprints =
```

Comment:
minor/shelve: we don't have tests yet, but we'll also need the `invenio-mail` and `invenio-celery` dependencies on the module.